frontend/util.lua

```diff
@@ -652,12 +653,12 @@ function util.getFriendlySize(size)
     size = tonumber(size)
     if not size or type(size) ~= "number" then return end
     local s
-    if size > 1024*1024*1024 then
-        s = string.format("%4.1f GB", size/1024/1024/1024)
-    elseif size > 1024*1024 then
-        s = string.format("%4.1f MB", size/1024/1024)
+    if size > 1073741824 then
+        s = string.format("%4.1f GB", rshift(size / 1024, 10) / 1024)
```

Comment:
I find these ones quite unreadable :) vs the clearer `size/1024/1024/1024`.
Given how non-performance impacting this function is (called in Book information and nothing much else), I think that optimisation vs clarity would not be really needed.
(If you insist, please put the simple/naive formula as a comment.)

Reply:
Yeah, I have to agree with that. Maintainability over performance, unless it's clear there's an issue. ;-)

Reply:
Fair enough ;).